packages/components/molecules/f-card-with-content/package.json

```diff
@@ -49,9 +49,7 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.2.0",
-    "@justeat/f-button": "3.x",
-    "@justeat/f-card": "3.x"
+    "@justeat/browserslist-config-fozzie": ">=1.2.0"
   },
   "devDependencies": {
     "@justeat/f-button": "3.3.0",
```

Comment:
My understanding of mixing the two types of dependencies is that it allows consuming projects to pass a package they already have installed into multiple dependant packages.

So `peerDependencies` allows consuming projects to pass a package version into multiple packages. Whereas `devDependencies` is there purely for when running this package in isolation i.e. testing or possibly Storybook? 

By removing `peerDependencies` are we not preventing consumers of the package from using whichever version they like and, possibly, allowing for duplicate versions of packages to be used? i.e. they might already use f-button v3.3.1 in their project but this package will bundle and use v3.0.0 alongside it.

Reply:
Yeh, so to confirm, the peerrDependency is there to make consumers explicitly aware that they need to use that version of a package alongside this one (so here, that is that f-button, f-card etc need to be installed as dependencies to use f-card-with-content).

The devDependency is solely there for testing as part of the unit tests and for storybook.

A potential solution to this could be to make the devDependency match the range of the peerDependency, so that the tests just run with the latest published version of the component (as essentially, they should run against this anyway, as that's what the peerDependency is saying). The only issue we could have with this is if someone manages to push a component change that does break a dependent test, but I don't see that as a downside, as we would want to know about that asap anyway?

Reply:
All fixed!